native/Avalonia.Native/src/OSX/WindowImpl.mm

```diff
@@ -208,6 +211,26 @@
     }
 }
 
+HRESULT WindowImpl::SetCanMinimize(bool value) {
+    START_COM_CALL;
+
+    @autoreleasepool {
```

Comment:
1) we probably don't need an ARP when we are on the UI thread, since there should be one provided by the event loop
2) use START_COM_ARP_CALL macro that enables both COM-safeguard and an ARP

Reply:
All other COM methods in this class were using ARP, so I followed.
I've replaced `START_COM_CALL` + `@autoreleasepool` with `START_COM_ARP_CALL`, as suggested.